packages/react-ui/src/Fields/Radio.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Label } from "./Label";
+import { animations, box, fonts, spacings } from "../theme";
+
+export const InputRadio = ({ label, name, id, ...props }) => (
+  <StyledLabel htmlFor={id}>
+    <StyledRadio type="radio" name={name} id={id} {...props} />
+    {label}
+  </StyledLabel>
+);
+
+InputRadio.propTypes = {
+  label: PropTypes.string.required,
+  name: PropTypes.string.required,
+  id: PropTypes.string.required
+};
+
+const RADIO_SIZE = "2rem";
```

Comment:
like Checkbox, a size props could be handy

Reply:
Done

Reply:
Done